### PR TITLE
BaSP-TG-130: Modal repeated email

### PR DESCRIPTION
**Pull request by maxidelfino**

### CODE REVIEW

File: src/Components/SuperAdmins/index.js [hunk 1]
```diff
@@ -30,7 +30,12 @@ const SuperAdmins = () => {
   };
 
   const onRowClick = (id) => {
-    history.push(`/super-admin/form/${id}`);
+    if (id === user._id) {
+      console.log('son iguales');
```
Comment (maxidelfino) [inline src/Components/SuperAdmins/index.js:34]: remove console.log

**Review by Licamus — APPROVED** [review]

File: src/Components/SuperAdmins/Form/index.js [hunk 2]
```diff
@@ -78,14 +78,30 @@ const Form = () => {
...
+    if (
+      getValues('name') &&
+      getValues('lastName') &&
+      getValues('email') &&
+      isEditing &&
+      !Object.values(errors).length
+    ) {
+      return (
+        <div>
+          <h4>Edit New Super Admin</h4>
```
Comment (visojulian) [inline src/Components/SuperAdmins/Form/index.js:102]:
```suggestion
          <h4>Edit Super Admin</h4>
```

File: src/Components/SuperAdmins/Form/index.js [hunk 3]
```diff
@@ -182,41 +198,47 @@ const Form = () => {
...
-          name="repeatPassword"
-          register={register}
-          type={reveal ? 'text' : 'password'}
-          placeholder="Repeat Password"
-          error={errors.repeatPassword?.message}
-        />
+          </>
+        ) : (
+          ''
+        )}
```
Comment (visojulian) [inline src/Components/SuperAdmins/Form/index.js:234]:
```suggestion
        )}
```

File: src/Components/SuperAdmins/Form/index.js [hunk 4]
```diff
@@ -182,41 +198,47 @@ const Form = () => {
...
-            placeholder="Password"
-            error={errors.password?.message}
-          />
-          <div className={styles.revealButton}>
-            <Button
-              text={reveal ? 'Hide password' : 'Reveal password'}
-              type="button"
-              variant="secondary"
-              onClick={showPassword}
+        {!isEditing ? (
```
Comment (visojulian) [inline src/Components/SuperAdmins/Form/index.js:201]:
```suggestion
        {!isEditing && (
```

**Review by camilavaleriarojas — APPROVED** [review]

**Pull request merged**